images/inventory.proto

```diff
@@ -21,4 +26,5 @@ message inventory_entry {
 	optional uint32			pre_dump_mode	= 9;
 	optional bool			tcp_close	= 10;
 	optional uint32			network_lock_method	= 11;
+	optional plugins_entry		plugins = 12;
```

Comment:
I think it should be just an array of strings. Plugins should be able to add entries on the dump stage and they should be able to delete entries on the restore stage. After processing all plugins, criu checks whether it is empty or not. If it isn't empty, criu reports a error that some plugins haven't been loaded? Does it make sense?

Reply:
It makes sense. Protobuf doesn't have `optional array` type but we can use `repeated string` instead.

>  After processing all plugins, criu checks whether it is empty or not.

Would it make sense to load only the plugins listed in the `plugins` field during restore? In other words, if the field is empty, we don't need to load any plugins.

Reply:
> Would it make sense to load only the plugins that are listed in the plugins field during restore? In other words, if the field is empty, we don't need to load any plugins.

It might better to make this on the plugin level. On restore, a plugin checks if it is in the list and makes a decision what hooks to register.

Reply:
Thanks @avagin! I've updated the pull request with these changes.